vms/platformvm/block/executor/standard_block_test.go

```diff
@@ -474,12 +480,11 @@ func TestBanffStandardBlockUpdateStakers(t *testing.T) {
 			},
 		},
 		{
-			description:   "advance time to staker5 end",
+			description:   "advance time to staker5 start",
 			stakers:       []staker{staker1, staker2, staker3, staker4, staker5},
 			advanceTimeTo: []time.Time{staker1.startTime, staker2.startTime, staker3.startTime, staker5.startTime},
 			expectedStakers: map[ids.NodeID]stakerStatus{
 				staker1.nodeID: current,
-				staker2.nodeID: current,
```

Comment:
Why is this changed in this PR? asserting `staker2.nodeID: current,` is correct based on how time is advanced in this test

Reply:
I felt it was weird.
We should not expect staker2 to be available when chainTime reaches its end time. In prod a ProposalBlock would be eventually issued to remove it from the validator set.
In the test we'll find it since we don't issue the proposalBlock and we just assert the advance time behaviour.
I reintroduced the staker with a comment.

Reply:
I don't think it is required for a proposal block to have been issued in this case. It's valid to issue a `StandardBlock` to advance the time up to the time to remove a validator (but not actually remove them).